### ISSUE TITLE
Add post-install disclaimer about plugin security

### ISSUE THREAD

**Issue by corneliusweig** [kubernetes-sigs/krew#315]
In SIG-CLI bi-weekly it was discussed that krew should add a notice to be shown after plugin installations. This note should clarify that plugins have not undergone a security validation and may be harmful. It is the users' responsibility to only run plugins from sources they trust.

**Comment by ahmetb** [comment]
+1
I think when we start supporting multiple indexes, we can potentially make it index-level (i.e. plugins you install from this index can be harmful, do you accept?) so that they're less annoying.

/priority important-soon

**Comment by corneliusweig** [comment]
I fully agree. With multiple repos we could even think about adding trust by using signed commits which get verified by git. But that's another story.
